packages/types/src/common/common.ts

```diff
@@ -191,3 +192,7 @@ export interface FindPaginationParams {
   offset?: number
   limit?: number
 }
+
+export interface MedusaRequest extends Request {
```

Comment:
comment: we already have that in the global types of medusa, should we move it? (`src/types/global.ts`)
```ts
declare global {
  // eslint-disable-next-line @typescript-eslint/no-namespace
  namespace Express {
    interface Request {
      user?: (User | Customer) & { customer_id?: string; userId?: string }
      scope: MedusaContainer
      validatedQuery: RequestQueryFields & Record<string, unknown>
      validatedBody: unknown
      listConfig: FindConfig<unknown>
      retrieveConfig: FindConfig<unknown>
      filterableFields: Record<string, unknown>
      allowedProperties: string[]
      includes?: Record<string, boolean>
      errors: string[]
    }
  }
}
```

Reply:
No, it is alright.
Just for some reason the default project is not picking that and the ts is failing. will close this PR.